### ISSUE TITLE
Infinity loading contract verification

### ISSUE THREAD

**Issue by bulbazavr1024** [blockscout/blockscout#7374]
### Environment

* Deployment type (Manual/Docker/Docker-compose): docker-compose
* Elixir & Erlang/OTP versions (`elixir -version`): docker-compose
* Node JS version (`node -v`): docker-compose
* Operating System: ubuntu 22.04 LTS
* Blockscout Version/branch/commit: master 
* Archive node type && version (Erigon/Geth/Nethermind/Ganache/?): Geth

Link to discussion: https://github.com/blockscout/blockscout/issues/5665

**Comment by bulbazavr1024** [comment]
@vbaranov Thank you for a fast answer! Yes, I have started all requirement services
1) Docker containers
<img width="1417" alt="image" src="https://user-images.githubusercontent.com/77885154/235483134-0bfa20ca-bd12-485f-b131-f5a1cb0f476a.png">
2) Smart-contract verification service logs
<img width="1069" alt="image" src="https://user-images.githubusercontent.com/77885154/235483305-c7826e6e-a7b8-4b8b-a10b-4e25b7e7f716.png">

**Comment by vbaranov** [comment]
@bulbazavr1024 if you start the verification process on the same contract 2nd time, will this issue appear again? Or it is only reproduced once?

**Comment by bulbazavr1024** [comment]
@vbaranov Yes. If I reload the page, and click verify - I have same problem ( infinity loading... ), but contract is verified.

**Comment by bulbazavr1024** [comment]
@vbaranov if it can help you, I can send you url of scanner or something else usefull information

**Comment by bulbazavr1024** [comment]
@vbaranov 
![image](https://github.com/blockscout/blockscout/assets/77885154/04c13a0d-c29a-455a-882c-088ee4f67fdd)
![image](https://github.com/blockscout/blockscout/assets/77885154/c8a8b80c-eafc-4abc-83ad-1730930b5406)
This is containers logs. My contract is verified but I have infinity loading button
![image](https://github.com/blockscout/blockscout/assets/77885154/8108eac6-d7be-4c6d-8050-66c140997059)

**Comment by Jaycelv** [comment]
> @vbaranov ![image](https://user-images.githubusercontent.com/77885154/242323882-04c13a0d-c29a-455a-882c-088ee4f67fdd.png) ![image](https://user-images.githubusercontent.com/77885154/242323932-c8a8b80c-eafc-4abc-83ad-1730930b5406.png) This is containers logs. My contract is verified but I have infinity loading button ![image](https://user-images.githubusercontent.com/77885154/242324154-8108eac6-d7be-4c6d-8050-66c140997059.png)

I met the same issue, have you resolved this problem?

**Comment by ReeRiley** [comment]
> Yes

**Comment by ReeRiley** [comment]
Yes

**Comment by nikitosing** [comment]
Should be fixed in new UI: https://github.com/blockscout/frontend
